test/verify/check-system-info

```diff
@@ -501,7 +504,7 @@ class TestSystemInfo(MachineCase):
 
         # Memory details should be shown for generic QEMU memory
         # demidecode not available on certain images
-        if m.image not in ['fedora-atomic']:
```

Comment:
this was not skipped on `rhel-atomic` before, but is now.

Reply:
Ah, but that is `rhel-7.7` based so we test it only in stable rhel-7 branches. Nevermind :)

Reply:
Right, I found it easier that way. Also, right now on master we don't test *any* atomic image any more. After this PR lands, I'll clean up the *-atomic special cases.